### PR TITLE
Api shipping methods unit tests

### PR DESCRIPTION
**Pull request by danac-gs**
# Description
Api/ShippingMethods.php unit test coverage

Fixes: https://app.asana.com/0/1131034199022718/1137538625478179/f

(Optional: Add a changelog by writing a comment after "#changelog" on the next line)

\#changelog

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have created or modified unit tests to sufficiently cover my changes.
- [x] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: Test/Unit/Model/Api/ShippingMethodsTest.php [hunk 1]
```diff
@@ -346,84 +366,165 @@ public function getShippingMethodsWithFullAddressData()
...
+        ];
+        $shippingAddress = [
+            'company'         => "",
+            'country'         => "United States",
+            'country_code'    => "US",
+            'email'           => "integration@bolt.com",
+            'first_name'      => "YevhenBolt",
+            'last_name'       => "BoltTest2",
+            'locality'        => "New York",
+            'phone'           => "+1 231 231 1234",
```
Comment (YevhenShBolt) [inline Test/Unit/Model/Api/ShippingMethodsTest.php]: @matt-thomason @danac-gs 
As i see last time Bolt have another format of phone.
It should be like `2312311234` without `+1` code

File: Test/Unit/Model/Api/ShippingMethodsTest.php [hunk 2]
```diff
@@ -610,56 +722,439 @@ public function shippingEstimationWithDiscountAppliedToShipping()
...
+        $email = "integration@bolt.com";
+        $shippingAddressData = [
+            'company'         => "",
+            'country'         => "United States",
+            'country_code'    => "US",
+            'email'           => $email,
+            'first_name'      => "John",
+            'last_name'       => "McCombs",
+            'locality'        => "Knoxville",
+            'phone'           => "+1 231 231 1234",
```
Comment (YevhenShBolt) [inline Test/Unit/Model/Api/ShippingMethodsTest.php]: The same as above regarding phone format.

File: Test/Unit/Model/Api/ShippingMethodsTest.php [hunk 3]
```diff
@@ -610,56 +722,439 @@ public function shippingEstimationWithDiscountAppliedToShipping()
...
```
Comment (YevhenShBolt) [inline Test/Unit/Model/Api/ShippingMethodsTest.php]: The same as above regarding phone format.

File: Test/Unit/Model/Api/ShippingMethodsTest.php (hunk 1 above)
Comment (danac-gs) [reply Test/Unit/Model/Api/ShippingMethodsTest.php]: done

File: Test/Unit/Model/Api/ShippingMethodsTest.php (hunk 2 above)
Comment (danac-gs) [reply Test/Unit/Model/Api/ShippingMethodsTest.php]: done

File: Test/Unit/Model/Api/ShippingMethodsTest.php (hunk 3 above)
Comment (danac-gs) [reply Test/Unit/Model/Api/ShippingMethodsTest.php]: done

**Comment by danac-gs** [comment]
@matt-thomason @njfaries please merge

**Pull request merged**